src/Form/Type/ChoiceMapper/AttributesMapper/AttributesTypeDateMapper.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * You can find more information about us on https://bitbag.io and write us
+ * an email on hello@bitbag.io.
+ */
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusElasticsearchPlugin\Form\Type\ChoiceMapper\AttributesMapper;
+
+class AttributesTypeDateMapper implements AttributesMapperCollectorInterface
+{
+    private const ATTRIBUTES_TYPE = 'date';
+
+    public function supports(string $type): bool
+    {
+        return self::ATTRIBUTES_TYPE === $type;
+    }
+
+    public function map(array $attributeValues): array
+    {
+        $choices = [];
+
+        foreach ($attributeValues as $productAttributeValue) {
+            $choice = $productAttributeValue['value']->format('d-m-Y H:i');
+            $value = $productAttributeValue['value']->format('Y-m-d H:i:s.sss');
```

Comment:
Is the `.sss` needed?

Reply:
In order not to file it's full date format, elasticsearch stores our date in this version 2017-01-01 00: 13: 00.000000